projects/fourier/lib/fourier/utilities/system.rb

```diff
@@ -9,12 +9,17 @@ def self.system(*args)
         Kernel.system(*args) || exit(1)
       end
 
-      def self.tuist(*args)
-        Dir.chdir(Constants::TUIST_DIRECTORY) do
-          self.system("swift", "build")
-          @tuist = ".build/debug/tuist"
-          self.system(@tuist, *args)
+      def self.tuist(*args, **kwargs)
+        @tuist = "/usr/local/bin/tuist"
```

Comment:
Using `@` you are making that a static variable in `System`. I'd update it to just `tuist`.